index.md

```diff
@@ -111,4 +111,9 @@ Want to see nmstate in action? Check out the [demos](./demos.md).
 
 ## Stack support:
 - [IPv4 addresses (static)](./examples.md#interfaces-ethernet)
+- [IPv6 addresses (static)](./examples.md#interfaces-ethernet)
```

Comment:
Pointing to the same location is weird, how about just editing line 113 with `IPv4/6 addresses ...`?

No need to specify the limitation of IPv6 here, you can add a note in the example itself. Same for the link-local info.
(We actually need a man or user guide to explain each property, the command line, the behaviour and so on. Unfortunately we do not have capacity for that at the moment.)